client/extensions/woocommerce/app/order/order-customer/create.js

```diff
@@ -0,0 +1,182 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { every, isEmpty } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import AddressView from 'woocommerce/components/address-view';
+import Button from 'components/button';
+import Card from 'components/card';
+import CustomerAddressDialog from './dialog';
+import { editOrder } from 'woocommerce/state/ui/orders/actions';
+import getAddressViewFormat from 'woocommerce/lib/get-address-view-format';
+import { getOrderWithEdits } from 'woocommerce/state/ui/orders/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import SectionHeader from 'components/section-header';
+
+class OrderCustomerInfo extends Component {
+	static propTypes = {
+		editOrder: PropTypes.func.isRequired,
+		orderId: PropTypes.oneOfType( [
+			PropTypes.number, // A number indicates an existing order
+			PropTypes.shape( { id: PropTypes.string } ), // Placeholders have format { id: 'order_1' }
+		] ).isRequired,
+		siteId: PropTypes.number.isRequired,
+	};
+
+	state = {
+		showDialog: false,
+	};
+
+	updateAddress = ( type = 'billing' ) => {
+		const { siteId, order } = this.props;
+		return address => {
+			const { copyToShipping = false, ...newAddress } = address;
+			if ( siteId ) {
+				this.props.editOrder( siteId, { id: order.id, [ type ]: newAddress } );
+				if ( copyToShipping && 'billing' === type ) {
+					this.props.editOrder( siteId, { id: order.id, shipping: newAddress } );
+				}
+			}
+		};
+	};
+
+	toggleDialog = type => {
+		return () => {
+			this.setState( { showDialog: type } );
+		};
+	};
+
+	renderDialogs = () => {
+		const { billing, shipping } = this.props.order;
+		return [
+			<CustomerAddressDialog
+				key="dialog-billing"
+				address={ billing }
+				closeDialog={ this.toggleDialog( false ) }
+				isBilling
+				isVisible={ 'billing' === this.state.showDialog }
+				updateAddress={ this.updateAddress( 'billing' ) }
+			/>,
+			<CustomerAddressDialog
+				key="dialog-shipping"
+				address={ shipping }
+				closeDialog={ this.toggleDialog( false ) }
+				isVisible={ 'shipping' === this.state.showDialog }
+				updateAddress={ this.updateAddress( 'shipping' ) }
+			/>,
+		];
+	};
+
+	renderBilling = ( address = {} ) => {
+		const { translate } = this.props;
+		if ( every( address, isEmpty ) ) {
+			return (
+				<Button
+					className="order-customer__add-link"
+					onClick={ this.toggleDialog( 'billing' ) }
+					primary
+				>
+					{ translate( 'Add Billing Address' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Billing Address' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).